### DISCUSSION TITLE
Hide a specific bundle and show it to specific user type

### DISCUSSION THREAD

**Discussion by Hadi-bakhshi** [webpack/webpack#15843] [General]
Hi,
I'm wondering if there's any way to have different bundles and use one of them if the admin logs in. Or if a user logs in there isn't a file like Admin.js bundle for security for instance.
Thanks in advance.

**Comment by alexander-akait** [comment]
I think better ask it on stackoverflow, thanks

**Reply by Hadi-bakhshi** [reply]
Got no answer there

**Reply by vankop** [reply]
you can have different webpack configs, since webpack config is js file you can implement this condition there, e.g.:
```js
if (isUserAdmin()) module.exports = config1;  else module.exports = config2;
```

**Reply by Hadi-bakhshi** [reply]
Thanks a million
